src/tools/cephfs/cephfs-shell

```diff
@@ -94,13 +94,18 @@ def to_bytes(string):
     return bytes(string, encoding='utf-8')
 
 
-def list_items(dir_name=''):
+def list_items(shell, dir_name=''):
```

Comment:
why is this shell parameter needed?

Reply:
The `shell` parameter is a left over of my debugging efforts that used the `self.poutput()` method of the shell. Since `list_items()` was a global function, it didn't have access to the shell object. Thinking back, I guess I could have used `sys.out.write()` as well. Do you want me to remove references to all such changes. They are all over the place in quite a few functions.